hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/BackupInfo.java

```diff
@@ -213,8 +213,8 @@ public void setType(BackupType type) {
     this.type = type;
   }
 
-  public void setBackupRootDir(String targetRootDir) {
-    this.backupRootDir = targetRootDir;
+  public void setBackupRootDir(String restoreRootDir) {
```

Comment:
same here. im guessing you jsut did a search/replace, but we'll need to make sure all of the changes make sense.